docs-source/content/userguide/managing-domains/model-in-image/model-files.md

```diff
@@ -93,7 +98,8 @@ During domain home creation, model, and property files are first loaded from the
 
 The loading order within each of these locations is first determined using the convention `filename.##.yaml` and `filename.##.properties`, where `##` are digits that specify the desired order when sorted numerically. Additional details:
 
- * Embedding a `.##.` in a filename is optional and can appear anywhere in the file name before the `properties` or `yaml` extension.
+ * Embedding a `.##.` in a filename is optional.
```

Comment:
file name (adj) (n) (Oops, looks like a per-exisitng error. Use two words in instructional text, but use filename as a variable in code instructions and commands.)